treeherder/webapp/api/performance_data.py

```diff
@@ -66,6 +66,11 @@ def list(self, request, project):
                 # test may be empty in case of a summary test, leave it empty
                 # then
                 ret[signature_hash]['test'] = test
+            else:
+                # if it's a summary test, we will query subtests signature for it
+                summary_signature = signature_data.get(signature_hash=signature_hash)
+                ret[signature_hash]['subtests'] = summary_signature.subtests.values_list(
+                    'signature_hash', flat=True).distinct()
```

Comment:
I'm pretty sure we don't need the `distinct()` here, as signature_hash should be distinct by definition.